backend/app/services/chatbot_conversation_service.py

```
@@ -291,34 +294,23 @@ async def continue_conversation(
                 f"User ID: {user_id} | Fallback response fetched successfully"
             )
 
-            logger.debug(f"Getting the chatbot prompt role")
+            logger.debug("Getting the chatbot prompt role")
             custom_prompt_role_of_chatbot = chatbot.get(
                 "role_of_chatbot"
             )
             
-            logger.debug(f"Getting the chatbot ai_model_name")
+            logger.debug("Getting the chatbot ai_model_name")
             ai_model_name = chatbot.get(
```

Ruff (0.8.2)

Local variable `ai_model_name` is assigned to but never used

Remove assignment to unused variable `ai_model_name`

(F841)